{{ cookiecutter.formal_name }}/app/src/main/res/drawable/splash_screen_background.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@color/colorPrimary"/>
```

Comment:
I'm guessing this color will need to be a variable that the Android backend exposes as a template option.

Reply:
Yeah -- I'm not sure how to thread this through in beeware land. Maybe something we can discuss.